packages/todos-lib/client.js

```diff
@@ -0,0 +1,11 @@
+/* global TodosLib:true */
+
+// I'm not sure exactly where this should go
+TodosLib = {
+  showError(error) {
+    if (error) {
+      // Use the global __ so that we can get symbols from other packages
```

Comment:
Oh, so `__` is scoped to the package we are in somehow. Of course `todo-lib` isn't defining any of the translations.